src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx

```diff
@@ -450,14 +450,15 @@ To change settings, you have two options (if the setting supports it):
         <tr>
             <td>
                 Enable or disable offline data storage when no internet connection is available.
-                
-                * To configure the amount of offline storage, see [Set max offline storage size](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-max-offline-storage/#android).
-                * Default value is `false`.
-                
+
+                To configure the amount of offline storage, see [Set max offline storage size](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-max-offline-storage).
+
             </td>
             <td>
+                Offline storage is disabled by default. To enable it, add the following feature flag:
+
                 ```java
-                NewRelic.disableFeature(FeatureFlag.OfflineStorage)
+                NewRelic.enableFeature(FeatureFlag.OfflineStorage)
```

Comment:
@ndesai-newrelic just to confirm, to enable offline storage for android, is it `NewRelic.disableFeature(FeatureFlag.OfflineStorage): false`, or `NewRelic.enableFeature(FeatureFlag.OfflineStorage)` ?

Reply:
just this is required NewRelic.enableFeature(FeatureFlag.OfflineStorage)